ui/webui/test/machine_install.py

```diff
@@ -173,7 +181,7 @@ def start(self):
     def kill(self):
         self._execute(f"virsh -q -c qemu:///session destroy {self.label} || true")
         self._execute(
-            f"virsh -q -c qemu:///session undefine "
+            f"virsh -q -c qemu:///session undefine --nvram "
```

Comment:
Please add a note comment why the --nvram option is needed - thanks! :)

Reply:
I added the comment. Without `--nvram` telling virsh to also remove the EFI NVRAM it created for the EFI VM the `undefine` call will fail to remove the VM.